File: app/src/main/java/cn/sichuan/cd/zsh/zsh/MyApplication.java
package cn.sichuan.cd.zzsy.zsh;

import cn.sichuan.cd.common.BaseCommonApplication;


public class MyApplication extends BaseCommonApplication {

    private static MyApplication instance;

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;


    }
    public static MyApplication getInstance() {
        return instance;
    }



}

File: app/src/main/java/cn/sichuan/cd/zsh/ui/MyApplication.kt
package cn.sichuan.cd.zzsy.ui

import com.blankj.utilcode.util.LogUtils
import cn.sichuan.cd.zzsy.model.app.base.BaseApplication

import com.scwang.smart.refresh.header.ClassicsHeader
import com.scwang.smart.refresh.layout.SmartRefreshLayout

class MyApplication : BaseApplication() {

    companion object {
        init {
            SmartRefreshLayout.setDefaultRefreshHeaderCreator { context, _ ->
                ClassicsHeader(context)
            }
        }

       lateinit var   instan:MyApplication;
    }


    override fun onCreate() {
        super.onCreate()
        instan=this;
        LogUtils.getConfig().run {
//            isLogSwitch = BuildConfig.DEBUG
            setSingleTagSwitch(true)
        }
    }
}